pkg/apps/application.scss

```diff
@@ -22,6 +22,21 @@
   align-items: center;
 }
 
+.progress-bar {
+  display: inline-block;
+  width: 20%;
+  overflow: hidden;
+}
+
+.sum-or-prog {
+  display: flex;
+  align-items: center;
+}
+
+.progress-title-span {
```

Comment:
Please remove this as well, also margin-right only works for LTR (Left to right) languages. For RTL languages like hebrew, this does not work, as the whole page is swaped in place.

See for example this margin missing in Hebrew.

![Screenshot from 2023-03-28 21-29-57](https://user-images.githubusercontent.com/14921356/228346928-165f528c-044c-4d57-986a-ffe5319cf5ec.png)

Reply:
By using the suggested Flex this will not just be fixed, but it's about to be fixed upstream in patternfly for the Flex component by using `gap` instead of `margin`